Sources/Moya/Plugins/AccessTokenPlugin.swift

```diff
@@ -64,6 +66,19 @@ public struct AccessTokenPlugin: PluginType {
         case .basic, .bearer:
             let authValue = authorizationType.rawValue + " " + tokenClosure()
             request.addValue(authValue, forHTTPHeaderField: "Authorization")
+          
+        case .urlParameter:
+          var existingComponents = URLComponents(url: request.url!, resolvingAgainstBaseURL: true)
```

Comment:
Any reason for the force unwrap here? We might want to crash more gracefully (or skip) if the url is not valid.

Reply:
Yeh, I agree. Would be better if I will use optional binding for url, and then continue other logic?